examples/js/controls/TransformControls.js

```diff
@@ -1307,6 +1311,14 @@ THREE.TransformControlsGizmo = function () {
 
 			}
 
+			// Hide disabled axes
+			for ( var a in handle.name ) {
```

Comment:
`lgtm.com` complains at this point since the loop variable `a` is not used in the loop body. Any chances to write this differently?

https://lgtm.com/projects/g/mrdoob/three.js/snapshot/7870913233927804e8a15595bc1d241b6f6504ed/files/examples/js/controls/TransformControls.js?#xfbc544f1af7ef263:1

Reply:
I have no idea why I put that in a loop. Silly me. I'll add a fix shortly.

Reply:
Fixed in #14770